### DISCUSSION TITLE
HA e scenari Alexa

### DISCUSSION THREAD

**Discussion by Ector73** [indomus/forum#2336] [HUB: Home Assistant]
Ciao a tutti, innanzitutto un doveroso ringraziamento per le fantastiche guide presenti sul sito: mi hanno aiutato tantissimo soprattutto a non desistere nelle difficoltà che inevitabilmente si incontrano se ci si addentra nella domotica casalinga.
Veniamo al punto: ho integrato con successo HA con Alexa (via haaska e AWS). Tutto liscio nell'installazione, nel test, nella creazione dei domini da esporre ad Alexa. Da alexa vedo i 2 scenari che ho creato su HA, ma se li eseguo con una routine da Alexa , su HA non succede nulla.

![image](https://user-images.githubusercontent.com/123939064/218447984-7541e338-dfbf-4e66-977b-ce47b6d05d6d.png)

Come potete vedere dall'immagine ho esposto due automazioni: se le eseguo da HA funzionano alla perfezione, ma su alexa non danno nessun esito.

![image](https://user-images.githubusercontent.com/123939064/218448113-4c1f5b6c-71b3-4720-840a-47538aa67b7a.png)

Su Alexa vedo lo scenario e il pulsante ON, ma sia deselezionando il pulsante che attivando lo scenario da routine non esegue l'automazione su HA.
Dove sbaglio? mi son perso qualche passaggio??

**Comment by Larry-78** [comment]
Ciao, non ho mai avuto la necessità di esporre automazioni quindi non posso verificarlo ma secondo me se fai in quel modo Alexa si limiterà ad attivare/disattivare le automazioni invece di farle eseguire.
Lo puoi verificare tenendo sotto controllo lo stato dell'entità automation.allarme_accendi_manuale in HA developer tools e provare a far partire la routine per vedere se attiva/disattiva l'automazione.

**Reply by Ector73** [reply]
..è proprio così,, la rotine di Alexa mi "abilita/disabilita" l'automazione ma non la esegue.
Perdonami..sono alle "prime armi"...hai idea di come posso ovviare?

**Reply by snakuzzo** [reply]
Non devi esporre un'automazione verso alexa...non ha alcun senso
Al massimo dovrai esporre uno **SCRIPT** (che sostanzialmente sarebbe composto dalla sola action che attualmente hai nell'automazione)

**Reply by Larry-78** [reply]
> ..è proprio così,, la rotine di Alexa mi "abilita/disabilita" l'automazione ma non la esegue. Perdonami..sono alle "prime armi"...hai idea di come posso ovviare?

... come immaginavo, l'ho sempre ritenuto inutile infatti snakuzzo conferma.

**Reply by Ector73** [reply]
GRAZIE!!...infatti usando uno script tutto funziona alla grande!
ps: un altro concetto appreso ;-)
